packages/lib/src/components/MBWay/components/MBWayInput/MBWayInput.tsx

```diff
@@ -40,7 +40,7 @@ function MBWayInput(props: MBWayInputProps) {
         <div className="adyen-checkout__mb-way">
             <Field
                 errorMessage={!!errors.telephoneNumber && i18n.get('telephoneNumber.invalid')}
```

Comment:
We should use `mobileNumber.invalid` instead of `telephoneNumber.invalid`.
Could you double check if we have that string in our translation files? If not, you'll need to add it in the base strings file and generate the translation files again.